test/run_test.py

```diff
@@ -427,17 +433,23 @@ def main():
         selected_tests = filter(lambda test_name: "jit" in test_name, TESTS)
 
     for test in selected_tests:
-        test_name = 'test_{}'.format(test)
+        splits = test.rsplit("/", 1)
```

Comment:
jit does this by adding a `test_jit.py` in the `torch/test` folder and then import other tests from there. I wonder if we should do the same.

cc @yf225 @ezyang @suo